src/ActionModal/utils/ActionManager.js

```diff
@@ -163,48 +76,40 @@ export default class ActionManager {
 
     let txMessages = []
     if (type === transaction.WITHDRAW) {
-      const validators = getTop5RewardsValidators(
-        context.bondDenom,
-        context.rewards
-      )
+      const validators = getTop5RewardsValidators(bondDenom, rewards)
       await Promise.all(
         validators.map(async validator => {
-          const txMessage = await transformMessage(
-            context.networkId,
-            type,
-            context.userAddress,
-            {
-              validatorAddress: validator
-            }
-          )
+          const txMessage = await getMessage(networkId, type, userAddress, {
+            validatorAddress: validator
+          })
           txMessages.push(txMessage)
         })
       )
     } else {
-      const txMessage = await transformMessage(
-        context.networkId,
+      const txMessage = await getMessage(
+        networkId,
         type,
-        context.userAddress,
+        userAddress,
         transactionProperties
       )
       txMessages.push(txMessage)
     }
 
-    const createSignedTransaction = await getTransactionSigner(context)
+    const createSignedTransaction = await signedTransactionCreator(networkId)
     const signedMessage = await createSignedTransaction(
       messageMetadata,
       txMessages,
       signer,
-      context.chainId,
-      context.account.accountNumber,
-      context.account.sequence
```

Comment:
So, I'm trying to get this PR. OK, the `Txservice` can now go to hell. But what's up with context? Why is it of no use now with transaction abstraction? I really want to understand this PR

Reply:
This PR is really just removing code and simplifying the existing code by removing some abstractions